lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -166,7 +156,7 @@ class TableSyntax extends BlockSyntax {
 
     for (var i = 0; i < row.length && i < alignments.length; i++) {
       if (alignments[i] == null) continue;
-      row[i].attributes['style'] = 'text-align: ${alignments[i]};';
+      row[i].attributes['align'] = '${alignments[i]}';
```

Comment:
It looks like `<td align` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-align) in favor of `<td style="text-align:`.

Reply:
Yes. it is. Not sure why Github spec has not updated it.